File: uri 1165 - Prime Number.py
n = int(input())
for i in range(n):
    x = int(input())
    flag = 0
    for j in range(2, x):
        if x % j == 0:
            flag = 1
            break
    if flag == 0:
        print(f"{x} eh primo")
    else:
        print(f"{x} nao eh primo")

File: uri 1165 - Prime Number.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int x,a,i,t;
    scanf("%d",&x);
    while(x>0){
        x--;
        i=2;
        t=0;
        scanf("%d",&a);
        while(i<a){
            if(a%i == 0 ){
                printf("%d nao eh primo\n",a);
                t=1;
                break;
            }
            i++;
        }
        if(t==0)
            printf("%d eh primo\n",a);

    }


    return 0;
}
